server/routes/additionalConditions.js

```diff
@@ -26,6 +26,15 @@ module.exports = function({logger, conditionsService, licenceService}) {
         logger.debug('POST /additionalConditions');
 
         const nomisId = req.body.nomisId;
+
+        const validationObject = await conditionsService.validateConditionInputs(req.body);
+        if (!validationObject.validates) {
+
+            const conditions = await conditionsService.getAdditionalConditionsWithErrors(req.body, validationObject);
+
+            return res.render('additionalConditions/index', {nomisId, conditions, submissionError: 'MISSING_INPUTS'});
+        }
+
         await licenceService.updateLicenceConditions(req.body);
```

Comment:
i'd have validationObject pass back a "clean" version of the data rather than using req.body after validating it